### ISSUE TITLE
unable to complete minimal steps

### ISSUE THREAD

**Issue by danshao** [micromdm/scep#65]
Building from source code:

```
./scepclient-darwin-amd64 -server-url=fff --private-key=./self.pem  -challenge=secret
ts=2017-10-13T06:38:47.992627Z level=info op=GetCACaps error="Get http://fff?operation=GetCACaps: dial tcp: lookup fff: no such host" took=8.846073ms
ts=2017-10-13T06:38:47.994655Z level=info op=GetCACaps error="Get http://fff?operation=GetCACaps: dial tcp: lookup fff: no such host" took=1.894425ms
unmatched type or headers
```

Running from release builds:

```
$ ./scepclient-darwin-amd64 -private-key client.key -server-url=http://localhost:2016 -challenge=secret
asn1: structure error: tags don't match (16 vs {class:0 tag:20 length:48 isCompound:true}) {optional:false explicit:false application:false defaultValue:<nil> tag:<nil> stringType:0 timeType:0 set:false omitEmpty:false} certificate @2
```

**Comment by groob** [comment]
> dial tcp: lookup fff: no such host"

Are you actually trying to connect to a server at `fff`? Not sure what I'm looking at here. 

The release error is because you need /scep in the url

**Comment by danshao** [comment]
I went by another thread using a local key. Perhaps you can update the readme? 

Sounds like I should be calling `./scepclient-darwin-amd64 -private-key client.key -server-url=http://localhost:2016/scep -challenge=secret`

**Comment by groob** [comment]
I merged #60 which had some updates for the readme. 

Does this resolve your issue?

**Comment by danshao** [comment]
Great, thanks!
